digitalaidseattle/games-for-love
# Mayk/update filter

## Description

This code change is includes updates related to my current sprint tasks, focusing on enhancing the sort and filter functionalities, updating the UI based on the wireframe, and improving map interactivity.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

[Update the funding values to percentage for sort functionality and display](https://projects.digitalaidseattle.org/project-tasks-details?recordId=rec522L7w6kBsoWY9)
[Update the filter logic for location-based queries](https://projects.digitalaidseattle.org/project-tasks-details?recordId=recNZqX5L0hWImNRc)
[Update the filter UI and functionality according to the wireframe](https://projects.digitalaidseattle.org/project-tasks-details?recordId=rec7pwTsFeLJLow5a)
[Map Interactivity](https://projects.digitalaidseattle.org/project-tasks-details?recordId=recy0iPeWxTFaODvy)
[Update Sort and Filter UI](https://projects.digitalaidseattle.org/project-tasks-details?recordId=recY0SHySKDW9bSZj)

added [Hospital Images on Interactive Map page](https://projects.digitalaidseattle.org/project-tasks-details?recordId=recJkoiC4uKiPLUHi)

## QA Instructions, Screenshots, Recordings

To test the funding value percentage update:

Verify the sort functionality displays funding values as percentages.
To test the filter logic for location-based queries:

Ensure location-based filters return the expected results.
To test the updated filter UI:

Confirm that the filter UI aligns with the provided wireframe and functions as expected.
To test map interactivity:

Test the map for interactive elements and confirm they function correctly.
To test the Sort and Filter UI updates:

Ensure the UI changes match the design specifications and operate smoothly for both sort and filter functionality.
